lib/method_object/version.rb

```diff
@@ -1,5 +1,5 @@
 # frozen_string_literal: true
 
 class MethodObject
-  VERSION = '2.1.5'
+  VERSION = '2.2.5'
```

Comment:
'2.2.0'?

Reply:
Yup. :) It's been awhile since I played with versions.